packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -171,7 +171,7 @@
   &__item {
     margin: rem-override(0.25rem);
   }
-  margin: 0px rem-override(-0.5rem);
+  margin: 0;
```

Comment:
I think that doing this in the formation override stylesheet might just temporarily adjust the margin. When we remove this stylesheet (after the base font size is updated in Formation for USWDS base font size), this update will eventually be removed with it. 

If this is a USWDS v3 override, it might be better to add it to the component stylesheet directly.